### ISSUE TITLE
default logger seems very uninformative?

### ISSUE THREAD

**Issue by berndbischl** [schalkdaniel/compboost#242]
i get this

```

library(mlbench)
data(Sonar)

cboost = Compboost$new(data = Sonar, target = "Class", loss = BinomialLoss$new())
cboost$addBaselearner(feature = c("V1"), id = "foo", PolynomialBlearner, degree = 1, intercept = TRUE)
cboost$addBaselearner(feature = c("V2"), id = "foo", PolynomialBlearner, degree = 1, intercept = TRUE)
cboost$addBaselearner(feature = c("V3"), id = "foo", PolynomialBlearner, degree = 1, intercept = TRUE)
cboost$addBaselearner(feature = c("V4"), id = "foo", PolynomialBlearner, degree = 1, intercept = TRUE)
cboost$train(3)

```

```
      Iteration |
-----------------
            1/3 |
            2/3 |
            3/3 |

```

Should I see like what is selected and what i the current loss?

**Comment by ja-thomas** [comment]
+1 for loss 

imo. the selected baselearner is not very useful as a direct trace since you're usually interested in the frequency (table) and order of occurrence (and if it was selected at all). All of that is hard to see from the raw trace, especially since iterations usually go by very quick

**Comment by berndbischl** [comment]
i dont care about the details but at least something shhould be there - on top of pure iteration numbers.
i thought daniel has so many flexible loggers. cant we simply configure a default one?

**Comment by schalkdaniel** [comment]
This is a default one, we register one iterations logger as default which logs the iterations. But we can improve the output by also printing the inbag risk or the selected base-learner. Should be easy. Another option would be to additionally register a time logger to print the elapsed time.

Maybe it would be nice to print, for example, just every 10th iteration?

**Comment by berndbischl** [comment]
yes 3x

**Comment by schalkdaniel** [comment]
Now it is possible to specify the steps how frequently we want to print something to the screen + a small info in the end (#252):
```r
cboost = boostSplines(data = iris, target = "Petal.Length", loss = LossQuadratic$new(), 
  iterations = 1000, trace = 250)
##    1/1000: risk = 1.4
##  250/1000: risk = 0.023
##  500/1000: risk = 0.021
##  750/1000: risk = 0.02
## 1000/1000: risk = 0.019
##
## Train 1000 iterations in 0 Seconds.
## Final risk based on the train set: 0.019
##
```
